mobile/src/screens/marketplace.js

```diff
@@ -418,6 +436,20 @@ class MarketplaceScreen extends Component {
         this.goBackToDapp()
       }
     }
+
+    if (url.hostname === 't.me') {
+      const success = await this.openNativeDeepLink(
+        url,
+        'lastOpenTelegramProfileTime',
+        true
+      )
+      if (success) {
+        return true
+      }
+    }
+
```

Comment:
While we are here we might as well solve #3310. Happy to do the work on that.

Reply:
#3310 might be a little tricky. We shouldn't break test-build links (under Developer's mode) and attestations (we redirect users for attestations on mobile).

Reply:
Good point, let’s leave that for a separate PR.

Reply:
Good point, let’s leave that for a separate PR.